tls/docs/kube-apiserver Client Certificates/README.md

```diff
@@ -409,6 +404,66 @@ Client certificate used by the network connectivity checker of the kube-apiserve
 | /etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/check-endpoints-client-cert-key/tls.crt/tls.key | -rw-------. | root | root | system_u:object_r:kubernetes_file_t:s0 |
 
 
+
+### system:serviceaccount:openshift-monitoring:prometheus-k8s
+![PKI Graph](subcert-systemserviceaccountopenshift-monitoringprometheus-k8s245984941195439209045265417880856366162.png)
+
+
+
+| Property | Value |
+| ----------- | ----------- |
+| Type | Client |
+| CommonName | system:serviceaccount:openshift-monitoring:prometheus-k8s |
+| SerialNumber | 245984941195439209045265417880856366162 |
+| Issuer CommonName | [kube-controller-manager-csr-signer-signer](#kube-controller-manager-csr-signer-signer) |
+| Validity | 23h |
+| Signature Algorithm | SHA256-RSA |
+| PublicKey Algorithm | ECDSA 256 bit, P-256 curve |
+| Usages | - KeyUsageDigitalSignature<br/>- KeyUsageKeyEncipherment |
+| ExtendedUsages | - ExtKeyUsageClientAuth |
+| Organizations (User Groups) |  |
+
+
+#### system:serviceaccount:openshift-monitoring:prometheus-k8s Locations
+| Namespace | Secret Name |
+| ----------- | ----------- |
+| openshift-monitoring | metrics-client-certs |
+
+| File | Permissions | User | Group | SE Linux |
+| ----------- | ----------- | ----------- | ----------- | ----------- |
+
+
+
+
+### system:serviceaccount:openshift-oauth-apiserver:openshift-authenticator
+![PKI Graph](subcert-systemserviceaccountopenshift-oauth-apiserveropenshift-authenticator122907677913153492574449624837636959338.png)
+
+
+
+| Property | Value |
+| ----------- | ----------- |
+| Type | Client |
+| CommonName | system:serviceaccount:openshift-oauth-apiserver:openshift-authenticator |
+| SerialNumber | 122907677913153492574449624837636959338 |
+| Issuer CommonName | [kube-controller-manager-csr-signer-signer](#kube-controller-manager-csr-signer-signer) |
+| Validity | 23h |
```

Comment:
23 hrs compared to 12 hrs elsewhere

Reply:
23h is not uncommon in general: https://github.com/openshift/api/pull/1244/files/22b40785b3c3e58c30a736e045625ccb93c07421#diff-6802f3f618ab81ff4b03c6eeec7abf71d28d10f15bf716e583f926bc0cabfca2L63.

Reply:
Some of the current 12h-certs were 23h-certs before.

Reply:
It seems to be an issue in displaying the values. They get rounded down:

```
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 6305382435363863953 (0x578137eae8d9bd91)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: OU=openshift, CN=kubelet-signer
        Validity
            Not Before: Aug  9 11:55:15 2022 GMT
            Not After : Aug 10 11:43:23 2022 GMT
        Subject: CN=kube-csr-signer_@1660046116
```